docs/UPGRADING.md

```diff
@@ -0,0 +1,38 @@
+# Upgrade guides
+
+This document includes the list of required changes when upgrading an existing JSS app.
+Each version contains the list of files, dependencies and environment variables that need to be applied or updated. Unreleased changes are listed in [JSS Next](#jss-next).
```

Comment:
a bit misleading for a first time reader. Can we just name it as Unreleased just like in our Changelog?